zero-module.yml

```diff
@@ -39,3 +39,6 @@ parameters:
   - field: stagingBackendSubdomain
     label: Staging Backend Host Name (e.g. api.)
     default: api.
+  - field: randomSeed
+    label: Random seed that will be shared between projects to come up with deterministic resource names
+    execute: uuidgen
```

Comment:
uuidgen actually gives a pretty long name, alot of AWS artifacts have character limit we might not be able to use it :(

Reply:
not sure what's a good alternative... `crc32 <(uuidgen)` ?

Reply:
Or just `uuidgen | head -c 8`